recipe/meta.yaml

```diff
@@ -0,0 +1,216 @@
+        - {{ compiler('cxx') }}
+      host:
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        - nomkl                                # [mxnet_blas_impl == 'openblas']
+        - cython
+        - python
+        # Numpy's C API is not used
+        - numpy >1.16.0,<=2.0.0
+        - requests >=2.20.0,<3
+        - setuptools
+      run:
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        - numpy >1.16.0,<=2.0.0
+        - requests >=2.20.0,<3
+        - python
+
+    test:
+      script: py-mxnet_test.py
+      imports:
+        - mxnet
+
+    about:
+      home: http://mxnet.io
+      license: Apache-2
+      license_file: LICENSE
+      license_family: Apache
+      summary: MXNet is a deep learning framework designed for both efficiency and flexibility
+      description: |
+        Apache MXNet (incubating) is a deep learning framework designed for both
+        efficiency and flexibility. It allows you to mix symbolic and imperative
+        programming to maximize efficiency and productivity. At its core, MXNet
+        contains a dynamic dependency scheduler that automatically parallelizes both
+        symbolic and imperative operations on the fly. A graph optimization layer on
+        top of that makes symbolic execution fast and memory efficient. MXNet is
+        portable and lightweight, scaling effectively to multiple GPUs and multiple
+        machines. MXNet is also more than a deep learning project. It is also a
+        collection of blue prints and guidelines for building deep learning systems,
+        and interesting insights of DL systems for hackers.
+      dev_url: https://github.com/apache/incubator-mxnet
+      doc_url: https://mxnet.incubator.apache.org/
+
+
+  - name: mxnet
+    version: {{ mxnet_version }}
+
+    build:
+      number: {{ build_number }}
+
+    requirements:
+      run:
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        - {{ pin_subpackage('py-mxnet', exact=True) }}
+
+    about:
+      summary: MXNet metapackage for installing lib,py-MXNet Conda packages
+
+
+  - name: mxnet-{{ mxnet_variant_str }}
+    version: {{ mxnet_version }}
+
+    #build:
+      #number: {{ build_number }}
+
+    requirements:
+      run:
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        - {{ pin_subpackage('_mutex_mxnet', exact=True) }}
+
+    about:
+      summary: MXNet metapackage which pins a variant of MXNet Conda package
+
+
+  - name: mxnet-gpu
+    version: {{ mxnet_version }}
+
+    build:
+      #number: {{ build_number }}
+      # Skip when the variant is NOT a gpu build.
+      skip: true                             # [not ('gpu' in str(mxnet_variant_str))]
+
+    requirements:
+      run:
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        - {{ pin_subpackage('_mutex_mxnet', exact=True) }}
+
+    about:
+      home: http://mxnet.io
+      summary: MXNet metapackage which pins a variant of MXNet(GPU) Conda package
+
```

Comment:
There seems to be a redirect for the `home_url` to the following location:
https://mxnet.apache.org/versions/1.9.1/

In case that we want to preserve the earlier we might benefit from using `https` instead of `http`

Reply:
I've updated these to `https://mxnet.apache.org`; however this still redirects to the latest version. I think - because these are simply the `home` tag - this is correct behavior.